server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -100,34 +105,36 @@ public Content render(
 
   private FormTag renderSearchForm(Http.Request request, SearchParameters searchParameters) {
     return form()
-        .withClass("w-1/4")
+        .withId("ti-search-form")
```

Comment:
Is there way to add a flash message or let the user know- "we can only search on complete dates" I think the TI will think, they can search by year for easier use.

Reply:
That's a good point.  Let me ask Sijia what they would recommend.

Reply:
Upon Sijia's recommendation, I added an error message when the dob is incomplete and the name field is empty.  The missing fields also have a red outline so that the user knows which ones are missing:
<img width="616" alt="image" src="https://github.com/civiform/civiform/assets/134556723/c4f9250a-c774-4412-83c9-82df12c4bbe4">

Reply:
beautiful!!!